desc/coils.py

```diff
@@ -1204,8 +1205,14 @@ def from_symmetry(cls, coils, NFP=1, sym=False):
             rotated_coils = coils.copy()
             rotated_coils.rotate(axis=[0, 0, 1], angle=2 * jnp.pi * k / NFP)
             coilset += rotated_coils
+        coilset_to_return = cls(*coilset)
+        if check_for_intersection:
```

Comment:
So this will always return the coilset regardless, this is just to possibly throw the warning?

Reply:
Yes